### ISSUE TITLE
Scroll in list of parameters for extensions isn't scrollable [Web app]

### ISSUE THREAD

**Issue by Bouh** [4ian/GDevelop#1476]
## Describe the bug
The scroll bar for function configuration in extension are not scrollable, only on web-app.

On screenshot for the function behavior, but it's same for classic function.
All tabs in **Function Configuration** are affected.

![image](https://user-images.githubusercontent.com/1670670/75562484-f30bbf80-5a48-11ea-997e-27d921cafd9c.png)

## Other details
- Only web app
- beta 88

**Comment by 4ian** [comment]
Damn, surely a flexbox issue where there is an extra/missing `flex`/`flex-basis`/`flex-grow`/`flex-shrink`. Always a bit of a pain to debug, but should be a question of time to find it with the Chrome debugger.

**Comment by deadpool1999** [comment]
May I please know which folder contains the code for this?

**Comment by Bouh** [comment]
Start your research with EventsFunctionConfigurationEditor component.

Le ven. 28 févr. 2020 à 19:05, deadpool1999 <notifications@github.com> a
écrit :

**Comment by jimil749** [comment]
Hello!

I did look into the issue and found the problem. Just had to provide predefined height to the outer container and that seemed to make the function configuration scrollable.
![Screenshot from 2020-02-29 15-11-27](https://user-images.githubusercontent.com/47107987/75605176-81924680-5b06-11ea-845f-5654ed2321c2.png)

I did this using Chrome dev tools.

**Comment by HarsimranVirk** [comment]
@4ian I have fixed this issue, should I open a PR or let others try?

**Comment by Bouh** [comment]
When you have fixed don't wait and open your PR :)

**Comment by 4ian** [comment]
Yes feel free to go ahead and make a PR => we'll discuss more in the PR if needed. :)
In particular, would be great to give this a try on Firefox/Chrome/Safari (if possible) and the Electron version (which is Chrome, but not the very latest version).

**Comment by 85ayush** [comment]
I would love to work on this issue, I am new to open source contribution, can you please guide me?
